internal/exec/util/unit.go

```diff
@@ -32,7 +31,7 @@ const (
 	DefaultPresetPermissions os.FileMode = 0644
 )
 
-func FileFromSystemdUnit(unit types.Unit, runtime bool) (*FetchOp, error) {
+func (ut Util) FileFromSystemdUnit(unit types.Unit, runtime bool) (*FetchOp, error) {
```

Comment:
Minor: should we either use `ut Util` in all functions and or `u Util` for consistency?